TestNumberPair.py

```diff
@@ -0,0 +1,14 @@
+from main import *
+from getColorPair import get_color_from_pair_number
+from PairNumberColor import get_pair_number_from_color
+
+def test_number_to_pair(pair_number,

```

Comment:
Nice separation of tests. Try to include the expectation in the name of the test. See if more test possibilities come up.